### ISSUE TITLE
unable to buy and sell something

### ISSUE THREAD

**Issue by flynk-zz** [peshay/btcde#48]
According to your example I tried to sell something with this codesnippet 

```
import btcde
create a object for the connection settings
api_key = "my_key"
api_secret = "my_secret"
conn = btcde.Connection(api_key, api_secret)
orderbook = conn.showOrderbook('sell', 'btceur')
print('API Credits Left: ' + str(orderbook.get('credits')))
orders = orderbook.get('orders')

for order in orders:
    print('Order ID: ' + str(order.get('order_id')) + " Min_Amount: " + str(order.get('min_amount')) + '\tPrice: ' + str(order.get('price')) + ' EUR')

conn.executeTrade('NRRHV9','sell','btceur',0.0021)`
```
Unfortunally it doesn't work for me.  I get this output.
```

WARNING:btcde:API Error Code: 5
WARNING:btcde:API Error Message: Invalid signature
WARNING:btcde:API Error URL: https://api.bitcoin.de/v2/trades/NRRHV9?order_id=NRRHV9&type=sell&trading_pair=btceur&amount=0.0021
```

Would be great if you could fix this?

**Comment by simone1999** [comment]
hello, have the same Problem. The Response is:
{"errors":[{"message":"Missing header: X-API-KEY","code":1,"field":"X-API-KEY"}]}
I think the API Key needs to be set in the X-API-KEY field of the request header.
interessting thou, that this seemd to have worked in the Past, but not annymore...

**Comment by simone1999** [comment]
hey, found the Problem:
the scipi (pip) version is not up to date with this Repo.
your error already got fixed in #44, but did not got pushed to scipy.
The workaround till it is getting pushed is to install it directly from git and not from scipy.
You can achife this with:
pip install git+git://github.com/peshay/btcde

**Comment by peshay** [comment]
I close this since there is workaround in place and I am working on a release of version 4.0 this should also end up in official pypi repo.
